src/bp/core/repositories/telemetry_payload.ts

```diff
@@ -0,0 +1,125 @@
+import { ConfigProvider } from 'core/config/config-loader'
+import { inject, injectable } from 'inversify'
+import _ from 'lodash'
+import moment from 'moment'
+
+import Database from '../database'
+import { TYPES } from '../types'
+
+const DEFAULT_ENTRIES_LIMIT = 1000
+
+interface TelemetryEntries {
+  url: string
+  events: any[]
+}
+
+interface TelemetryEntry {
```

Comment:
Can this be moved to the common typing as it can be useful for front end code as well

Reply:
done in b8a3b92b8f43967e36943f3b06e2b3c0ceb8596b